app/models/experimentation/treatment_group.rb

```diff
@@ -0,0 +1,11 @@
+module Experimentation
+  class TreatmentGroup < ActiveRecord::Base
+    belongs_to :experiment
+    has_many :reminder_templates
+    has_many :treatment_group_memberships
+    has_many :patients, through: :treatment_group_memberships
+
+    validates :index, presence: true, numericality: {greater_than_or_equal_to: 0}
```

Comment:
Index should also be unique within the experiment, correct?

Reply:
Yes. I was planning to write more model validations while I write a script for setting up experiments from a hash.

Reply:
follow up PR